crates/block-producer/src/runner.rs

```diff
@@ -400,6 +400,17 @@ impl BaseInitComponents {
 }
 
 pub fn run(config: Config, skip_config_check: bool) -> Result<()> {
+    // Set up sentry.
+    let _guard = match &config.sentry_dsn.as_ref() {
+        Some(sentry_dsn) => sentry::init((
+            sentry_dsn.as_str(),
+            sentry::ClientOptions {
+                release: sentry::release_name!(),
+                ..Default::default()
+            },
+        )),
+        None => sentry::init(()),
```

Comment:
IMO, ```None``` is better then ```sentry::init(())```.

Reply:
Actually we used sentry-log as a wrapper here. I tried None instead of sentry::init() before. It turned out that the sentry had to be initialized here even if we don't enable sentry at all. Because env_logger is wrapped into it and we have to init env_logger by init sentry log. If we set None here, then we cannot get any printed log.